freediscovery/stop_words.py

```diff
@@ -5,12 +5,204 @@
+"pearlie","zelma","malinda","noreen","tameka","saundra","hillary","amie",
+"althea","rosalinda","jordan","lilia","alana","gay","clare","alejandra",
+"elinor","michael","lorrie","jerri","darcy","earnestine","carmella","taylor",
+"noemi","marcie","liza","annabelle","louisa","earlene","mallory","carlene",
+"nita","selena","tanisha","katy","julianne","john","lakisha","edwina",
+"maricela","margery","kenya","dollie","roxie","roslyn","kathrine","nanette",
+"charmaine","lavonne","ilene","kris","tammi","suzette","corine","kaye","jerry",
+"merle","chrystal","lina","deanne","lilian","juliana","aline","luann","kasey",
+"maryanne","evangeline","colette","melva","lawanda","yesenia","nadia","madge",
+"kathie","eddie","ophelia","valeria","nona","mitzi","mari","georgette",
+"claudine","fran","alissa","roseann","lakeisha","susanna","reva","deidre",
+"chasity","sheree","carly","james","elvia","alyce","deirdre","gena","briana",
+"araceli","katelyn","rosanne","wendi","tessa","berta","marva","imelda",
+"marietta","marci","leonor","arline","sasha","madelyn","janna","juliette",
+"deena","aurelia","josefa","augusta","liliana","young","christian","lessie",
+"amalia","savannah","anastasia","vilma","natalia","rosella","lynnette",
+"corina","alfreda","leanna","carey","amparo","coleen","tamra","aisha","wilda",
+"karyn","cherry","queen","maura","mai","evangelina","rosanna","hallie","erna",
+"enid","mariana","lacy","juliet","jacklyn","freida","madeleine","mara","hester",
+"cathryn","lelia","casandra","bridgett","angelita","jannie","dionne","annmarie",
+"katina","beryl","phoebe","millicent","katheryn","diann","carissa","maryellen",
+"liz","lauri","helga","gilda","adrian","rhea","marquita","hollie","tisha",
+"tamera","angelique","francesca","britney","kaitlin","lolita","florine",
+"rowena","reyna","twila","fanny","janell","ines","concetta","bertie","alba",
+"brigitte","alyson","vonda","pansy","elba","noelle","letitia","kitty","deann",
+"brandie","louella","leta","felecia","sharlene","lesa","beverley","robert",
+"isabella","herminia","terra","celina"
 ]
+
+
+class _StopWordsWrapper(object):
+    """A mechanism for adding / managing custom stop words
+        Parameters
+        ----------
+        cache_dir : str
+           folder where the model will be saved
+        stop_words : list
+           a list of strings
+    """
+    _wrapper_type = "stop_words"
+
+    def __init__(self, cache_dir='/tmp/'):
+        self.cache_dir = cache_dir   # the path to dir
+
+    def save(self, name, stop_words):
+        """Allow to save the stop_words list of strings with joblib.save
+           under $CACHE_DIR/stop_words/<name>.pkl
+        """
+        self.name = name # the name (tag) for custom stop words list
+        self.stop_words = stop_words
+        if os.name == 'nt':
+            self.path_to_sw_dir = self.cache_dir + 'stop_words\\'
+        else:
+            self.path_to_sw_dir = self.cache_dir + 'stop_words/'
```

Comment:
You could just use `os.path.join(self.cache_dir, 'stop_words')` to make it cross OS compatible.. and let's call it `self.model_dir` (even if it's not a model) to be consistent with other wrappers...